docs/tutorials/user-transpiled-circuits.ipynb

```diff
@@ -31,7 +31,7 @@
        "  <rdf:RDF xmlns:dc=\"http://purl.org/dc/elements/1.1/\" xmlns:cc=\"http://creativecommons.org/ns#\" xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\">\n",
```

Comment:
<p>"store the results to <code>results</code>." &gt; "store the results to <code>result</code>."  because you used <code>result</code> as variable name in the code cell bellow. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
You might need to click "not trusted" at the top of the notebook app to allow the images